charts/telegraf/README.md

```diff
@@ -1,71 +1,66 @@
+   ```
 
-To install the chart with the release name `telegraf`:
+2. Run the following command, providing a name for your Telegraf release:
 
-```console
-helm upgrade --install telegraf influxdata/telegraf --namespace monitoring
-```
+   ```bash
+   helm upgrade --install my-release influxdata/telegraf
+   ```
 
-The command deploys Telegraf on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+   > **Tip**: `--install` can be shortened to `-i`.
 
-> **Tip**: List all releases using `helm list`
+   This command deploys Telegraf on the Kubernetes cluster using the default configuration. To find parameters you can configure during installation, see [Configure the chart](#configure-the-chart).
 
-## Uninstalling the Chart
+  > **Tip**: To view all Helm chart releases, run `helm list`.
 
-To uninstall/delete the `telegraf` deployment:
+## Uninstall the chart
 
-```console
-helm uninstall telegraf
+To uninstall the `my-release` deployment, use the following command:
+
+```bash
+helm uninstall my-release
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+This command removes all Kubernetes components associated with the chart and deletes the release.
 
-## Configuration
+## Configure the chart
 
-The default configuration parameters are listed in `values.yaml`.
+Plugins are configured as arrays of key/value dictionaries. Find configurable parameters, their descriptions, and their default values stored in `values.yaml`.
 
-```console
-helm upgrade --install telegraf influxdata/telegraf
-```
+To configure the chart, do either of the following:
 
-> **Tip**: `helm upgrade --install [RELEASE] [CHART] [FLAGS]` can be shortened : `helm upgrade -i [RELEASE] [CHART] [FLAGS]`
+- Specify each parameter using the `--set key=value[,key=value]` argument to `helm upgrade --install`. For example:
 
-Outputs and inputs are configured as arrays of key/value dictionaries. Additional examples and defaults can be found in [values.yaml](values.yaml)
+  ```bash
+  helm upgrade --install my-release \
+    --set persistence.enabled=true \
+      influxdata/telegraf
+  ```
 
-Example:
+  This command enables persistence.
 
-```yaml
-outputs:
-  - influxdb:
-      urls: []
-        # - "http://influxdb.monitoring:8086"
-      database: "telegraf"
-inputs:
-  - cpu:
-      percpu: false
-      totalcpu: true
-  - system:
-```
+- Provide a YAML file that specifies the parameter values while installing the chart. For example, use the following command:
+
+  ```bash
+  helm upgrade --install my-release -f values.yaml influxdata/telegraf
+  ```
 
-> **Tip**: You can use the default [values.yaml](values.yaml)
+  > **Tip**: Use the default [values.yaml](values.yaml).
 
-Please see https://github.com/influxdata/telegraf/tree/master/plugins/ and checkout the contents of the `inputs` and `outputs` folders.
```

Comment:
Are Telegraf aggregator and processor plugins configured differently for helm charts than inputs and output plugins?

Reply:
I don't believe so, no

Reply:
removed note